src/Game/AccountManagement.java

```diff
@@ -18,14 +18,23 @@
  */
 package Game;
 
+import java.math.BigInteger;
 import Client.CRC16;
 import Client.Logger;
 import Client.Settings;
 import Client.Util;
-import java.math.BigInteger;
 
 public class AccountManagement {
 
+  public static int panelPasswordChangeMode;
+  public static String oldPassword;
+  public static String newPassword;
+  public static int customQuestionEntry = -1;
+  
+  public static String recoveryQuestions[] = new String[]{"Where were you born?", "What was your first teachers name?", "What is your fathers middle name?", "Who was your first best friend?", "What is your favourite vacation spot?", "What is your mothers middle name?", "What was your first pets name?", "What was the name of your first school?", "What is your mothers maiden name?", "Who was your first boyfriend/girlfriend?", "What was the first computer game you purchased?", "Who is your favourite actor/actress?", "Who is your favourite author?", "Who is your favourite musician?", "Who is your favourite cartoon character?", "What is your favourite book?", "What is your favourite food?", "What is your favourite movie?"};
+  public static int recoveryIndices[] = new int[] { 0, 1, 2, 3, 4 };
+
+
   // § SECTION stream IO interaction §
 
   public static void register(String user, String pass) {

```

Comment:
Should note above this that this is function is intended to be RSC127 compatible

Reply:
added